go.sum

```diff
@@ -1,12 +1,34 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
you ran `go mod tidy`? this is a lot of new sum lines

Reply:
I ran it a few times, but not for the last commit, but running it on this commit is adding more lines as well (I'll commit them now).

Reply:
Ran it again.